src/app/public/modules/link-list/link-list.component.html

```diff
@@ -0,0 +1,15 @@
+<ng-container *ngIf="links.length > 0">
+  <h2 class="sky-font-heading-4 sky-margin-stacked-xs">
+    {{ title }}
+  </h2>
+  <ul class="link-list sky-margin-stacked-xl">
+    <li *ngFor="let link of links" class="sky-margin-stacked-xs">
```

Comment:
this spacing should be `sky-margin-stacked-sm` once that flows through from the design tokens update (same for recently accessed)